BookingProject/BookingWebAPI/BookingWebApi/Program.cs

```diff
@@ -108,16 +114,22 @@
 builder.Services.AddScoped<IAuthenticationService, AuthenticationService>();    
 builder.Services.AddScoped<IApartamentService, ApartamentService>();
 builder.Services.AddScoped<IApartamentStatisticService, ApartamentStatisticService>();
+builder.Services.AddScoped<IAppUserService, AppUserService>();  
 
 //Repositories
 builder.Services.AddScoped<IAppUserRepository, AppUserRepository>();
 builder.Services.AddScoped<IApartamentRepository,ApartamentRepository>();
-
+builder.Services.AddScoped<IAppUserRepository, AppUserRepository>();
 
 //Decorators
 builder.Services.AddScoped<IUserManagerDecorator<AppUser>, UserManagerDecorator<AppUser>>();
 builder.Services.AddScoped<ISignInManagerDecorator<AppUser>, SignInManagerDecorator<AppUser>>();
 
+//Kafka
+builder.Services.Configure<KafkaSettings>(builder.Configuration.GetSection("KafkaSettings"));
+builder.Services.AddSingleton<IBaseKafkaProducer<string,string>, UserChangeKafkaProducer>();
```

Comment:
What will be the signature of another producer u want to add? Probably the same)
Add additional interface for UserChangeKafkaProducer so that u can inject it more convenient